internal/app/ui/cpwsarea/wsmap/tools/fill.go

```diff
@@ -69,13 +70,32 @@ func (t *ToolFill) onStop(util.Point) {
 		return
 	}
 
+	bordersOnly := imguiext.IsCtrlDown()
```

Comment:
You can move that check right into `if` statement. It's a little be confused to see it here, since at that moment no prefab can be selected and var won't ever be used.

Reply:
done